frame/treasury/src/lib.rs

```diff
@@ -108,14 +109,15 @@ pub trait Trait: frame_system::Trait {
 	type Event: From<Event<Self>> + Into<<Self as frame_system::Trait>::Event>;
 
 	/// Handler for the unbalanced decrease when slashing for a rejected proposal.
-	type ProposalRejection: OnUnbalanced<RingNegativeImbalance<Self>>;
+	type RingProposalRejection: OnUnbalanced<RingNegativeImbalance<Self>>;
+	type KtonProposalRejection: OnUnbalanced<KtonNegativeImbalance<Self>>;
 
 	/// Fraction of a proposal's value that should be bonded in order to place the proposal.
 	/// An accepted proposal gets these back. A rejected proposal does not.
 	type ProposalBond: Get<Permill>;
 
 	/// Minimum amount of funds that should be placed in a deposit for making a proposal.
-	type ProposalBondMinimum: Get<RingBalance<Self>>;
+	type ProposalBondMinimum: Get<StakingBalanceT<Self>>;
```

Comment:
Maybe need a better way to declare `PropoosalBondMinimum`?...haven't figure out the logic between `ExistentialDeposit`, `Get`, and `parameter_types` currently now, but it seems the `Get<StakingBalanceT<Self>>` way is not correct.

Reply:
This requires to modify the source code of [`impl<T: Trait> Currency<T::AccountId> for Module<T>`][1]

[1]: https://github.com/darwinia-network/darwinia/blob/a61fbb51b3538ca19aa8d34c528ad0f3c2c3e1ce/frame/balances/kton/src/lib.rs#L479